package-res/reportviewer/reportviewer-prompt.js

```diff
@@ -344,14 +344,18 @@ define([
       showMessageBox: function( message, dialogTitle, button1Text, button1Callback, button2Text, button2Callback, blocker ) {
         var messageBox = registry.byId('messageBox');
 
-        messageBox.setTitle(dialogTitle);
-        messageBox.setMessage(message);
+        messageBox.setTitle('<div class="pentaho-reportviewer-errordialogtitle">' + dialogTitle + '</div>');
+        messageBox.setMessage('<div class="pentaho-reportviewer-errordialogmessage">' + message + '</div>');
 
```

Comment:
Are you sure this is correct? If you'd wanted to add a style class, I would expect that you set it in the template, not here as part of the string.